foursight_core/es_connection.py

```diff
@@ -165,17 +165,32 @@ def search(self, search, key='_source'):
         if err_msg:
             raise ElasticsearchException(message=err_msg)
         # In next line, PyCharm's linter wrongly worries that 'res' might not be reliably set above. -kmp 6-Jun-2022
-        return [obj[key] for obj in res['hits']['hits']] if len(res['hits']['hits']) > 0 else []  # noQA
+        total = res['hits']['total']
+        return [obj[key] for obj in res['hits']['hits']] if len(res['hits']['hits']) > 0 else [], total  # noQA
 
-    def get_result_history(self, prefix, start, limit):
+    def get_result_history(self, prefix, start, limit, sort = "timestamp.desc") -> [list, int]:
         """
         ES handle to implement the get_result_history functionality of RunResult
         """
+        sort_field = "uuid"
+        sort_order = "desc"
+        if sort:
+            if sort.endswith(".desc"):
+                sort_order = "desc"
+                sort_field = sort[:-5]
```

Comment:
I'm not a big fan of random numeric constants. You can get `remove_suffix` from `dcicutils.misc_utils` and then do:
```suggestion
                sort_field = remove_suffix(".desc", sort_field)
```
I'm even OK with making this a constant as in
```
SORT_DESC_SUFFIX = ".desc"
SORT_ASC_SUFFIX = ".asc"
```
so you can do:
```
if sort.endswith(SORT_DESC_SUFFIX):
    sort_order = 'desc'
    sort_field = remove_suffix(SORT_DESC_SUFFIX, sort)
...etc.
```
Or that might be overkill. But the reason for liking variables here is that it's harder for the two to slip out of sync with one another. One reason the integers are bad is if someone changes the spelling to be `DSC` or something like that, then the constant just randomly breaks.  It also means tests can be sure they're testing the right string.